packages/editor/CodeMirror/markdown/decoratorExtension.ts

```diff
@@ -10,6 +10,8 @@ import { ViewPlugin, DecorationSet, ViewUpdate } from '@codemirror/view';
 import { ensureSyntaxTree } from '@codemirror/language';
 import { RangeSetBuilder } from '@codemirror/state';
 
+const baseLineDecoration = Decoration.line({ class: 'CodeMirror-line' });
```

Comment:
```diff
- <div class="cm-line">...</div>
+ <div class="cm-line CodeMirror-line">...</div>
```

- [ ] Check which plugins need this.

Reply:
I've removed the `CodeMirror-line` class &mdash; RichMarkdown only needs it in 1-2 places.

Other plugins like ReMoods seem to use it, but also rely on other CSS classes being present.